bosh/releases/pre_render_scripts/scheduler/ssh_proxy/jobs/patch_ssh_proxy_ctl.sh

```diff
@@ -1,28 +0,0 @@
-#!/usr/bin/env bash
-
-set -o errexit -o nounset
-
-target="/var/vcap/all-releases/jobs-src/diego/ssh_proxy/templates/ssh_proxy_ctl.erb"
```

Comment:
Appears to be unused.

Reply:
Squashed history and amended commit message based on these comments.
Commit 5f7cd28c48738106936b93d1ae215ead9c8d412d